opentech/apply/funds/models.py

```diff
@@ -413,13 +414,42 @@ def build_json_order_by(field):
 
 
 class ApplicationSubmission(AbstractFormSubmission):
+    field_template = 'funds/includes/submission_field.html'
+
     form_data = JSONField(encoder=DjangoJSONEncoder)
+    form_fields = StreamField(CustomFormFieldsBlock())
```

Comment:
Would appreciate a discussion around this. 

It is the easiest in terms of rendering the form later but does rely on a definition for the field remaining in the repo. We could perhaps fall back to a Charfield if no field can be found.

Reply:
Just to confirm, we discussed having the fallback to Charfield and keeping things simple.